ros2_generate_interface_docs/ros2_generate_interface_docs/templates/srv.html.em

```diff
@@ -0,0 +1,56 @@
+@{
+import html
+import time
+}@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <title>@(interface_name) Documentation</title>
+    <link type="text/css" rel="stylesheet" href="../../msg-styles.css" />
+  </head>
+  <body>
+  <div id="container">
+    <h1 class="msg-title"><a href="../index-msg.html">@(interface_package)</a>/@(interface_name) @(type)</h1>
+    <div class="msg">
+      <h2>File: <span class="filename">@(interface_package)/@(interface_name).@(ext)</span></h2>
+      <h2>Raw Message Definition</h2>
+      <div class="raw-msg">
+      @[for line in raw_text.splitlines()]@
```

Comment:
The same comment about the indentation as on the previous PRs apply here.

Reply:
indentation https://github.com/ros2/ros2_generate_interface_docs/pull/6/commits/9d858279deb0f4d59567aaef64ff857cc7e59898